less/home-page.less

```diff
@@ -3,25 +3,31 @@
   .content-section {
     margin: 30px 0;
     display: flex;
-    flex-wrap: wrap;
+    flex-wrap: nowrap;
```

Comment:
Why did you change it to nowrap here?  It's making the home page not lining up properly